presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,68 @@
+* Fix query failures caused by Parquet predicate pushdown for smallint and tinyint types.
+* Fix query failures when view columns were fully qualified with catalog and/or schema.
+* Fix query failures when creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that would disable a performance optimization if two tables with mismatching
+  but compatible bucket counts are joined and one of the table has a `$bucket` filter.
+* Fix an issue that may cause procedure calls to fail if no queries were run after the server started up.
+* Fix an issue where the analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
+* Add :func:`ST_Length` for spherical geography.
+* Add ``view_owner`` column to the ``information_schema.views`` system table.
+* Add a ``JSON`` version of the query plan to ``QueryCompletedEvent``.
+* Add support for creating warnings during parsing.
+* Add a warning for using the ``current_role`` reserved word as an identifier.
+* Add support for using the empty string as a delimiter for the :func:`split` function.
+  When an empty string is used as the delimiter, the string will be split into individual characters.
+
+Raptor Changes
+--------------
+* Add ``raptor.minimum-node-count`` configuration property. The the number of nodes in a cluster is fewer
+  than the configured amount, data recovery and reassignment will not run.
+* Add configuration property ``raptor.startup-grace-period`` to control the delay of the initial bucket balancer run after the coordinator startup.
+
+Hive Connector Changes
+----------------------
+
+* Add ``hive.ignore-table-bucketing`` configuration property and ``ignore_table_bucketing`` session property.
+  When set to true, these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
+
+Verifier Changes
+----------------
+* Fix an issue where the determinism check for ``INSERT`` queries were not run when there was a result mismatch.
+* Fix an issue where the checksum query was treated as the main query, and the main query was treated as part of the setup queries,
+  for ``INSERT`` query verification.
+* Fix query timeout enforcement by replacing local timer with session property ``query_max_execution_time``.
+* Improve result comparison for floating point columns by using relative errors.
+  Replace configuration property ``expected-double-precision`` with ``relative-error-margin``.
+* Improve result comparison for orderable array columns by applying :func:`array_sort` before :func:`checksum`.
+* Improve result comparison for ``SELECT`` queries by rewriting ``SELECT`` queries as ``CREATE_TABLE_AS_SELECT`` and using checksum queries to verify the results.
+  This eliminates the row count limit for ``SELECT`` queries.
+* Improve Verifier interfaces to allow easy extension by community users.
```

Comment:
Do we really need: Improve Verifier interfaces to allow easy extension by community users. 
If we have docs for the new interfaces or a doc for how to extend the verifier I think we can keep this and link to the docs, otherwise I vote for removing this item as this is a general statement and doesn't tell something specific to the users.